src/rgw/rgw_admin.cc

```diff
@@ -229,8 +229,8 @@ void _usage()
   cout << "   --reset-regions           reset regionmap when regionmap update\n";
   cout << "   --bypass-gc               when specified with bucket deletion, would trigger\n";
```

Comment:
this text needs to be moved into the imperative or present tense "when specified with bucket deletion, triggers"--can't be "would"